### PR TITLE
RSLC- 9 As a system, I should delete targets after one week 

### PR DESCRIPTION
**Pull request by rodriguevara**
#### Board:
* [Ticket #9](https://trello.com/c/RVz0DUwz/10-rslc-9-as-a-system-i-should-delete-targets-after-one-week)
---
#### The system deletes the targets created after a week
---

### CODE REVIEW

File: config/initializers/scheduler.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'rufus-scheduler'
+
+scheduler = Rufus::Scheduler.singleton
+
+scheduler.every '1d' do
+  @expired_targets = Target.where('created_at < ?', 7.days.ago)
+  @expired_targets.each(&:destroy!)
```
Comment (enzofab91) [inline config/initializers/scheduler.rb]:
```suggestion
  expired_targets = Target.where('created_at < ?', 7.days.ago)
  expired_targets.each(&:destroy!)
```

Comment (enzofab91) [reply config/initializers/scheduler.rb]: This is fine, just to let you know that you can use a scope https://guides.rubyonrails.org/active_record_querying.html#scopes
And wrap the where condition in the model.

**Review by enzofab91 — APPROVED** [review]
LGTM

**Review by santiagodiaz — APPROVED** [review]
great work! 👏

**Pull request merged**